site/src/templates/standard-article.js

```diff
@@ -141,6 +141,27 @@ export default class Article extends React.Component {
     })
   }
 
+  concatenateRoles(roles) {
+    return roles.join(", ")
+  }
+
+  getRoles(post) {
+    return this.hasRoles(post)
+      ? this.concatenateRoles(post.frontmatter.roles)
+      : ""
+  }
+
+  hasRoles(post) {
```

Comment:
Can you not just use optional chaining syntax here?
return !!post?.frontmatter?.roles

or for more clarity

return post?.frontmatter?.roles ? true : false

Reply:
This is a proposal feature of ES8 and hasn't been accepted yet